README.md

```diff
@@ -101,5 +101,35 @@ You can use [django-apptemplates](https://github.com/bittner/django-apptemplates
 
 ---
 
+## Contributors
+
+### Code Contributors
+
+This project exists thanks to all the people who contribute. [[Contribute](CONTRIBUTING.md)].
```

Comment:
**Contribute** link is broken. The `CONTRIBUTING.md` file doesn't exist and it is not included in this PR.